### ISSUE TITLE
[ADD] Chaplain to VA service taxonomy

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#19895]
### Which Taxonomy is this term for?

VA services taxonomy

### Proposed term name/label

Chaplain services

### Proposed machine name/ API ID

chaplain

### Does this overlap with any existing term in the taxonomy?

No

### Rationale for adding this term

The possibility of adding this service was initially discussed in summer 2023.
Chaplain services are an important part of the healthcare journey for Veterans and their family members. 186 VAMC detail pages have been created for Chaplain services. Many sites also include stories featuring chaplains and their contributions. 

[Link to draft service](https://prod.cms.va.gov/health-care/chaplain-services)

### Which products will this term impact?

VAMCs

### Have stakeholders from impacted products been consulted?

Yes

### Have you drafted content for all the relevant subfields?

Yes

### Acceptance Criteria

**Content Review and Entry**
- [x] Term is reviewed by relevant taxonomy governance body
- [x] Term is either approved for addition, or denied with an explanation
- [x] If approved, a Drupal admin from Product team may add the term to the taxonomy and save as Draft
- [x] Content for subfields is updated in Drupal by Product team
- [ ] (List any other necessary review and approval steps here)

**Technical Coordination**
- [x] Work with Lighthouse to update the Facilities API to accept new value
  - This step can begin as soon as the API ID has been finalized and saved as Draft in Drupal 

**Launch & Change Management**
- [x] Once Content Review and Technical Coordination are complete, term can be published in Drupal
- [ ] Updates to KB articles and other editor training materials
- [ ] Announcement to editors

**Comment by davidmpickett** [comment]
> We should add steps to this template to reflect Lighthouse and change management tasks

Started a draft update: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/19909/files

**Comment by davidmpickett** [comment]
@mmiddaugh From a content perspective, this first sentence has a lot happening

> VA Chaplains address the spiritual, emotional, moral, religious, and ethical concerns of religious and non-religious inpatient and outpatient Veterans, their caregivers, and their families.

Consider breaking it up into 2-3 sentences:

VA Chaplains address spiritual, emotional, moral, religious, and ethical concerns of VA patients. Their services are available to religious and non-religious inpatient and outpatient Veterans, their caregivers, and their families.

**Comment by jilladams** [comment]
Added into sprint 18 for reporting purposes.

**Comment by mmiddaugh** [comment]
@davidmpickett agree with your feedback to break up the sentence making it easier to parse - thank you

**Comment by davidmpickett** [comment]
**End of Sprint update:**
- I provided content feedback on the draft term
- I updated the issue template to reflect technical coordination and launch steps
- Unassigning myself as others are handling technical coordination and launch

@jilladams @Agile6MSkinner

**Comment by jilladams** [comment]
ID was provided to LH and we asked about their timing to add to API [here](https://dsva.slack.com/archives/C02BTJTDFTN/p1733783753629569?thread_ts=1733259911.791349&cid=C02BTJTDFTN) in Slack.

**Comment by jilladams** [comment]
LH planning to get into final sprint before code freeze, that finishes 12/18.

**Comment by jilladams** [comment]
Chaplain service has been added to LH: https://dsva.slack.com/archives/C02BTJTDFTN/p1734468459796459?thread_ts=1733259911.791349&cid=C02BTJTDFTN
> chaplain has been added as a health service to the API and this has been deployed to Production [edit: and Sandbox] so it is available for use as soon as you start pushing those services.

Michelle published the term: https://prod.cms.va.gov/taxonomy/term/1190/revisions

I've updated the VHA DM trello card for this, so we can discuss next week. @mmiddaugh not sure about KBs. I'm going to go ahead and close this ticket, but we can always reopen if there's work remaining to do on that.
